### ISSUE TITLE
Docker image digest incorrect at docker hub

### ISSUE THREAD

**Issue by crogre** [adoptium/adoptium-support#684]
### Please provide a brief summary of the bug

I'm using the `eclipse-temurin:11.0.17_8-jre` image for linux/amd64 from docker hub. According to the tags page, https://hub.docker.com/_/eclipse-temurin/tags?page=1&name=11.0.17_8-jre, the digest of that image should be `Digest:sha256:772b0000ef09d2099ef07d1df87b12d942248419e79ff46fa739a5feb617e543`. However, the image I pulled has `Digest:sha256:81b132098ddaba7dc53b67e23e83452f458029a8eeafde911a3076f82d22e946`.

### Please provide steps to reproduce where possible

Just do a 
`docker pull eclipse-temurin:11.0.17_8-jre`

### Expected Results

Image digest should match the sha256 on docker hub.

### Actual Results

```
$ docker inspect eclipse-temurin:11.0.17_8-jre -f "{{.RepoDigests}}"

[eclipse-temurin@sha256:81b132098ddaba7dc53b67e23e83452f458029a8eeafde911a3076f82d22e946]
```

### What Java Version are you using?

Not applicable

**Comment by karianna** [comment]
@gdams This is a DockerHub only concern or on us?

**Comment by gdams** [comment]
What architecture are you pulling?

**Comment by crogre** [comment]
Amd64

**Comment by gdams** [comment]
Yeah, I can recreate this on my end. @tianon / @yosifkit any idea what might be going on here?
